src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -28,7 +28,8 @@ public SortCommand(String field) {
     public CommandResult execute(Model model) {
         requireNonNull(model);
         model.sortPetList(field);
-        return new CommandResult(MESSAGE_SUCCESS);
+        model.updateFilteredPetList();
+        return new CommandResult(MESSAGE_SUCCESS + field);
```

Comment:
Not sure if this is the problem but I see that in `SortCommandTest.java`, the success message seems to be missing the `field`, which could be why the program is failing the sort test cases.